src/Periphery/TokenWrapper.sol

```diff
@@ -18,10 +18,13 @@ interface IWrapper {
 /// @title TokenWrapper
 /// @author LI.FI (https://li.fi)
 /// @notice Provides functionality for wrapping and unwrapping tokens
-/// @custom:version 1.1.0
+/// @custom:version 1.2.0
 contract TokenWrapper is WithdrawablePeriphery {
     uint256 private constant MAX_INT = 2 ** 256 - 1;
```

Comment:
unused, can be removed

Reply:
Fixed dcb8a6d1: Removed unused MAX_INT constant